src/routes/safe/components/Apps/components/ConfirmTransactionModal.tsx

```diff
@@ -85,6 +86,7 @@ const ConfirmTransactionModal = ({
   onClose,
   onTxReject,
 }: OwnProps): React.ReactElement | null => {
+  const { nativeCoin } = getNetworkInfo()
```

Comment:
call outside the component (I'm thinking that this can be easily converted to a `NATIVE_COIN` const and would make things easier. But, by the time being, it's OK to just move it outside.